src/test/java/com/google/javascript/gents/singleTests/fields.js

```diff
@@ -11,6 +11,11 @@ var A = function(a) {
   let y = 1;
   /** @type {number} */
   this.z = y + 1;
+
+  this.w;
+  this.w.bar = 'bar';
+
+  this.n = 13;
```

Comment:
can you add a test where the constructor mentions `this.n` twice, e.g. assigns `12` into it on the next line?